tdrs-backend/reports/zap.conf

```diff
@@ -0,0 +1,3 @@
+10036	IGNORE	(Server leaks version information via Server HTTP Response Header field)
+10038	IGNORE	(Content security Policy (CSP) Header not set)
```

Comment:
@riatzukiza @abottoms-coder, there's an open issue (#907) to address the CSP owasp finding, so I didn't think we wanted to flag and ignore this one as a false-positive. Is there new information available about this that we should reconsider?